docs/src/content/docs/quickstart/sql-overview.mdx

```diff
@@ -0,0 +1,636 @@
+SELECT * FROM INFORMATION_SCHEM.columns`} />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+```sql
+user=> SELECT * FROM INFORMATION_SCHEMA.columns;
+ column_name |                                                                  data_type                                                                  | table_catalog | table_name | table_schema
+-------------+---------------------------------------------------------------------------------------------------------------------------------------------+---------------+------------+--------------
+ tx_time     | [:timestamp-tz :micro "UTC"]                                                                                                                | xtdb          | txs        | xt
+ committed   | :bool                                                                                                                                       | xtdb          | txs        | xt
+ _id         | :i64                                                                                                                                        | xtdb          | txs        | xt
+ error       | [:union #{:null :transit}]                                                                                                                  | xtdb          | txs        | xt
+ _id         | :i64                                                                                                                                        | xtdb          | people     | public
+ name        | :utf8                                                                                                                                       | xtdb          | people     | public
+ likes       | [:union #{[:list [:union #{:f64 :utf8 [:struct {dynamic :utf8}]}]] :null}]                                                                  | xtdb          | people     | public
+ info        | [:union #{[:struct {contact [:union #{:null [:list [:struct {loc :utf8, registered [:union #{[:date :day] :null}], tel :utf8}]]}]}] :null}] | xtdb          | people     | public
+(8 rows)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+Captureing fast-changing data may be powerful, but what if we made a mistake somewhere and wanted to undo a change? The thing that makes XTDB _most_ interesting is the approach to immutability and time-travel...
+
+## Query the past
+
+XTDB automatically records all versions and changes to individual rows across your database. This is widely useful for building reliable information systems, and it all starts with a [log](https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying).
+
+### A log of transactions
+
+Alongside an HTTP API, XTDB supports a [Postgres wire protocol](/drivers/postgres/getting-started) compatibility endpoint that enables developers to re-use existing tools and drivers that were built for connecting to real Postgres servers.
```

Comment:
I'm inclined to remove reference to the `HTTP API` here.
I think it's one thing too much